src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -73,13 +92,25 @@ private static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Pre
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
 
+    private static Optional<Prefix> getMissingPrefix(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
+        return Stream.of(prefixes)
+                .filter(prefix -> argumentMultimap.getValue(prefix).isEmpty())
+                .findFirst();
+    }
+
     /**
      * Returns true if there are duplicate prefixes(except for tag) in the given {@code ArgumentMultimap}.
      */
-    private static boolean checkDuplicatePrefixes(ArgumentMultimap argumentMultimaps) {
+    private static boolean checkDuplicatePrefixes(ArgumentMultimap argumentMultimap) {
```

Comment:
I guess we can move this to the `Parser` interface, since we will be using this method for checking prefixes on the other add/edit commands as well

Reply:
Move into `ParserUtils`

Reply:
Track in separate issue

Reply:
https://github.com/AY2021S1-CS2103T-F11-1/tp/issues/178